blenderproc/python/types/StructUtility.py

```diff
@@ -101,7 +101,8 @@ def get_all_cps(self) -> list:
 
         :return: A list of key value pairs
         """
-        return self.blender_obj.items()
+        cps = dict([pair for pair in self.blender_obj.items()])
+        return cps
```

Comment:
This can be one line

```suggestion
        return dict([pair for pair in self.blender_obj.items()])
```